ubuntu_install_requirements.sh

```diff
@@ -18,7 +18,6 @@ sudo apt -y update
 # Install required packages
 
 sudo apt -y install \
-  python-pip \
```

Comment:
Where does pip3 get installed now this is removed?

Reply:
I checked that pyenv is installing the pip3 so we don't need it